Sources/Redis/Client/RedisClient.swift

```diff
@@ -39,25 +47,30 @@ public final class RedisClient {
             }
         }
     }
+
+    /// Closes this client.
+    public func close() {
+        channel.close(promise: nil)
+    }
 }
 
 /// MARK: Config
 
 /// Config options for a `RedisClient.
-struct RedisClientConfig: Codable {
+public struct RedisClientConfig: Codable {
     /// Default `RedisClientConfig`
     public static func `default`() -> RedisClientConfig {
-        return .init(hostname: "localhost", port: 6379)
+        return .init(hostname: "127.0.0.1", port: 6379)
```

Comment:
we should keep this `localhost`, as that's how it appears in all other packages

Reply:
I was getting connection issues with localhost. I will try it again